nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -189,145 +134,48 @@ where
-                    txn.execute(query).await?;
+                // Commit transaction and relinqish the lock.
+                multi_txn.commit().await?;
 
-                    // This statement will update whether the creator is verified and the `seq`
-                    // number.  `seq` is used to protect the `verified` field, allowing for `mint`
-                    // and `verifyCreator` to be processed out of order.
-                    let mut query = asset_creators::Entity::insert_many(db_creator_verified_infos)
-                        .on_conflict(
-                            OnConflict::columns([
-                                asset_creators::Column::AssetId,
-                                asset_creators::Column::Creator,
-                            ])
-                            .update_columns([
-                                asset_creators::Column::Verified,
-                                asset_creators::Column::Seq,
-                            ])
-                            .to_owned(),
-                        )
-                        .build(DbBackend::Postgres);
-                    query.sql = format!(
-                        "{} WHERE excluded.seq > asset_creators.seq OR asset_creators.seq IS NULL",
-                        query.sql
-                    );
-                    txn.execute(query).await?;
-                }
+                // Upsert into `asset_creators` table.
+                upsert_creators(
+                    txn,
+                    id_bytes.to_vec(),
+                    &metadata.creators,
+                    slot_i,
+                    seq as i64,
+                    false,
+                )
+                .await?;
 
                 // Insert into `asset_authority` table.
-                let model = asset_authority::ActiveModel {
-                    asset_id: Set(id_bytes.to_vec()),
-                    authority: Set(bundle.keys.get(0).unwrap().0.to_vec()), //TODO - we need to rem,ove the optional bubblegum signer logic
-                    seq: Set(seq as i64),
-                    slot_updated: Set(slot_i),
-                    ..Default::default()
-                };
-
-                // Do not attempt to modify any existing values:
-                // `ON CONFLICT ('asset_id') DO NOTHING`.
-                let query = asset_authority::Entity::insert(model)
-                    .on_conflict(
-                        OnConflict::columns([asset_authority::Column::AssetId])
-                            .do_nothing()
-                            .to_owned(),
-                    )
-                    .build(DbBackend::Postgres);
-                txn.execute(query)
-                    .await
-                    .map_err(|db_err| IngesterError::AssetIndexError(db_err.to_string()))?;
+                //TODO - we need to remove the optional bubblegum signer logic
+                let authority = bundle.keys.get(0).unwrap().0.to_vec();
```

Comment:
Will be updated after https://github.com/metaplex-foundation/blockbuster/pull/31 is merged.